src/assets/i18n/en-us.json

```diff
@@ -220,5 +225,30 @@
     "price": "Fee",
     "fee": "Gas",
     "totalCost": "Total Cost"
+  },
+  "wallets": {
+    "wallets": "Wallets",
+    "pullToRefreshBalance": "Pull to refresh balance",
+    "assetWallet": "Asset Wallet",
+    "myAssetWallet": "My Asset Wallet",
+    "myAssetWalletTooltip": "Main asset vault for Capture NFT and NUM",
+    "integrityKey": "Integrity Key",
+    "integrityWallet": "Integrity Wallet",
+    "integrityWalletTooltip": "Wallet used for creating Signatures",
+    "numTransactionHistory": "Transaction History",
+    "viewOnBscScan": "View on BscScan",
+    "exportIntegrityKey": "Export Integrity Key",
+    "transfer": "Transfer",
+    "withdraw": "Withdraw",
+    "deposit": "Deposit",
+    "inAppTransfer": "In App Transfer",
+    "enterAmount": "Enter Amount",
+    "moveNumBetweenAssetWallet": "Move NUM between asset wallet",
+    "andIntegrityWallet": "and integrity wallet",
+    "requestSent": "request sent",
+    "gasFee": "Gas Fee",
+    "pending": "Pending",
+    "total": "Total",
+    "quoteGasFee": "Quote Gas Fee"
```

Comment:
The "Quote Gas Fee" doesn't sound very natural to me. Is this text an agreed discussion result? If yes, I have no problem. If not, I'd suggest double check if the wording are ok